kasa/smart/smartdevice.py

```diff
@@ -110,20 +135,10 @@ async def update(self, update_children: bool = True):
         for module in self.modules.values():
             req.update(module.query())
 
-        resp = await self.protocol.query(req)
+        self._last_update = resp = await self.protocol.query(req)
 
         self._info = self._try_get_response(resp, "get_device_info")
-
-        self._last_update = {
-            "components": self._components_raw,
```

Comment:
I don't think it's necessary to keep components around in the internal state after everything is initialized?